inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java

```diff
@@ -563,6 +564,7 @@ private static List<FieldInfo> parseFieldInfos(List<StreamField> streamFields, S
      */
     private static Map<String, String> parseProperties(Map<String, Object> properties) {
         return properties.entrySet().stream()
+                .filter(v -> Objects.nonNull(v.getValue()))
```

Comment:
Suggest use `StringUtils.isNotBank()`.

Reply:
> Suggest use `StringUtils.isNotBank()`.

Value is of type Object，so i use Objects.nonNull(Object obj)